src/pulse3D/plate_recording.py

```diff
@@ -746,7 +764,7 @@ def _load_optical_file_attrs(sheet: Worksheet):
     well_name = _get_excel_metadata_value(sheet, WELL_NAME_UUID)
 
     attrs = {
-        FILE_FORMAT_VERSION_METADATA_KEY: "N/A",
+        FILE_FORMAT_VERSION_METADATA_KEY: NOT_APPLICABLE_LABEL,
```

Comment:
do we need this at all if it's always not applicable?

Reply:
Yes it will still get loaded into the metadata page